mockup/patterns/structure/README.rst

```diff
@@ -0,0 +1,108 @@
+How to customize the Structure Updater pattern
+==============================================
+
+The ``mockup-pattern-structureupdater`` updates the title and description on the page - if available - when a folder is changed in the structure pattern. It's triggered by the class ``template-folder_contents``, which is available on the body tag when the folder contents page is opened. The pattern listens on the ``context-info-loaded`` event, which is triggered by the structure pattern.
+
+Often you will have customized layouts, where you need to update also other parts of the site when the user changes to another folder.
+
+
+Pattern configuration
+---------------------
+
+For simple cases, you can customize the CSS selector for the title and description via the options ``titleSelector`` and ``descriptionSelector`` for the pattern.
+This can be done by adding options for that pattern via the resource registry control panel under the "Pattern options" tab or via the ``registry.xml`` profile like so::
+
+  <record name="plone.patternoptions">
+    <value purge="False">
+      <element key="structureupdater">{"titleSelector": "h1.documentFirstHeading", "descriptionSelector": "footer"}</element>
+    </value>
+  </record>
+
+.. note::
+    For the ``titleSelector`` and ``descriptionSelector`` you have to provide valid JQuery selectors.
+    Like with any CSS selector you can also specify mutliple selectors by seperating them via a comma sign.
+
+
+Pattern overloading
+-------------------
+
+If you need some more control, you can overload the pattern and provide your own.
+
+The pattern is registered in the RequireJS configuratio in ``mockup/mockup/js/config.js`` under the name ``mockup-patterns-structureupdater`` and under the path ``patterns/structure/pattern-structureupdater``.
```

Comment:
the config.js path is mockup/js/config.js, right? mockup is there twice. One can assume that you mean inside the repository.